src/com/company/FootballMatch.java

```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class FootballMatch extends Service {
+
+    public int MatchForecast(int firstRealScore, int secondRealScore, int firstForecastScore, int secondForecastScore) {
+        return ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore)) ? 2 :
+        ((firstRealScore >= secondRealScore && firstForecastScore >= secondForecastScore)
+                || (firstRealScore <= secondRealScore && firstForecastScore <= secondForecastScore)) ? 1 : 0;
```

Comment:
if you write MatchForecast(2, 2, 3, 2) you must get result = 0
this code will work correctly:
return ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore)) ? 2 :
        ((firstRealScore > secondRealScore && firstForecastScore > secondForecastScore)
                || (firstRealScore < secondRealScore && firstForecastScore < secondForecastScore))
                || (firstRealScore == secondRealScore && firstForecastScore == secondForecastScore)) ? 1 : 0;

Reply:
Sofiia, I have some problems with github (sorry :) ), my PR is closed (can't fix it), so the PR is not updated. Should I create a new PR?